doc/rados/configuration/msgr2.rst

```diff
@@ -95,10 +95,10 @@ Similarly, two options control whether IPv4 and IPv6 addresses are used:
    not yet tested as of Nautilus v14.2.0 and likely needs some
    additional code changes to work correctly.
 
-Connection modes
-----------------
+Authentication modes
+--------------------
 
-The v2 protocol supports two connection modes:
+The v2 protocol supports two authentication modes:
```

Comment:
I doubt this replacement is an improvement. These modes are far more than just authentication (they include e.g. data intgrity0. I propose to leave this intact or dissect into a separated PR.